makefile

```diff
@@ -15,8 +16,18 @@ server: ## build the container
 	--env FLASK_ENV=production \
 	-d todo-app:prod
 
+test: ## kill the container
```

Comment:
We should call out this new addition in our README, to let users know they can run the tests in this manner (and also fix this copy-pasted comment!)